AGPs/AGP-CRDAO.md

```diff
@@ -0,0 +1,60 @@
+---
+AGP: N/A
+Title:  Community Rewards DAO
+Author: Aaron Foster (`@pythonpete32`) & Burrrata (`@burrrata`)
+Status: Stage III
+Track: Finance
+Created: 2019-10-01
+---
+
+# AGP X: Community Rewards DAO
+
+## Address of the transfer recipient
+
+TBD, but will be the Vault of the CRDAO
+
+## Amount of the transfer
+
+12,000 ANT
+- 1000 ANT for every week between this upcoming AGP (fourth Thursday of October) and the next AGP (fourth Thursday of January)
```

Comment:
The AA won't be sending weekly transactions, just one payment would be made and the CRDAO would need to handle weekly allocations to the DAO themselves

Reply:
Thank you for the feedback. Updated to a single transfer.